documents/processor.go

```diff
@@ -139,17 +169,23 @@ func (dp defaultProcessor) AnchorDocument(ctx context.Context, model Model) erro
 		return errors.New("failed to get document root: %v", err)
 	}
 
-	anchorID, err := anchors.ToAnchorID(model.CurrentVersion())
+	anchorID, err := anchors.ToAnchorID(model.CurrentVersionPreimage())
```

Comment:
Please rename:
```suggestion
	anchorIDPreImage, err := anchors.ToAnchorID(model.CurrentVersionPreimage())
```

The preImage is not the anchorID. 

The anchorID == cd.CurrentVersion

Otherwise it is confusing for the reader :)